source/quickstart.txt

```diff
@@ -1,12 +1,235 @@
 .. _c2c-quickstart:
 
-===========
-Quick Start
-===========
+==========
+Quickstart
+==========
 
 .. include:: /includes/preview-warning
 
 .. default-domain:: mongodb
 
-Get going!
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+{+c2c-full-product-name+} connects MongoDB clusters and provides a way
+to synchronize data between them. The tool that makes the connection is
+:ref:`c2c-mongosync`. This page provides a short introduction to help
+you get started with {+c2c-product-name+}. For more details, please see
+the rest of the {+c2c-product-name+} documentation.
+
+Setup
+-----
+
+{+c2c-product-name+} synchronizes data between two clusters. To run
+``mongosync``, you will need to: 
+
+- Define a source and a destination cluster.
+- Define administrative users 
+- Download and install ``mongosync``.
+- Connect the clusters.
+
+Follow the instructions below to setup {+c2c-product-name+}.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Define a source and a destination cluster
+
+      If you already have a MongoDB cluster, either self-managed or
+      hosted in :atlas:`MongodDB Atlas </getting-started?jmp=docs>`, 
+      use that cluster as the source cluster. If you don't have a
+      cluster to work with, you will need to create one.
+
+      You will need a destination cluster that has the same topology
+      as your source cluster. If the source cluster is :ref:`sharded
+      <sharding-background>`, the destination cluster must be sharded
+      in the same way. If the source cluster is a :ref:`replica set
+      <replication>` with 3 nodes, the destination cluster must also be
+      a replica set with 3 nodes.
```

Comment:
Suggestion flipping the statements so that the replica set goes first. 

The replica set statement is not true: 
If the source cluster is a replica set with 3 nodes, the destination cluster must also be a replica set but can have the same or a different number of nodes.